packages/core/src/browser/addEventListener.ts

```diff
@@ -90,7 +89,7 @@ type EventMapFor<T> = T extends Window
  * * returns a `stop` function to remove the listener
  */
 export function addEventListener<Target extends EventTarget, EventName extends keyof EventMapFor<Target> & string>(
-  configuration: Configuration,
+  configuration: { allowUntrustedEvents?: boolean | undefined },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ 
```suggestion
  configuration: { allowUntrustedEvents?: boolean },
```

or 

```suggestion
  configuration: { allowUntrustedEvents: boolean | undefined },
```

Reply:
I reused the same type definition as the InitConfiguration params https://github.com/DataDog/browser-sdk/blob/f5b9b0667fc72853347c4f59398eb53c62b78ad9/packages/core/src/domain/configuration/configuration.ts#L61-L62.
Basically we want customer to be able to do:
- `{ allowUntrustedEvents: undefined }`
- `{ allowUntrustedEvents: false }`
- `{ }`